src/main/kotlin/me/gserv/fabrikommander/commands/RtpCommand.kt

```diff
@@ -0,0 +1,93 @@
+package me.gserv.fabrikommander.commands
+
+import me.gserv.fabrikommander.data.PlayerDataManager
+import me.gserv.fabrikommander.data.spec.Pos
+import me.gserv.fabrikommander.utils.Context
+import me.gserv.fabrikommander.utils.Dispatcher
+import me.gserv.fabrikommander.utils.gold
+import me.gserv.fabrikommander.utils.gray
+import me.gserv.fabrikommander.utils.yellow
+import me.gserv.fabrikommander.utils.plus
+import me.gserv.fabrikommander.utils.aqua
+import net.minecraft.server.command.CommandManager
+import net.minecraft.util.Identifier
+import net.minecraft.server.world.ServerWorld
+import net.minecraft.util.registry.Registry
+import net.minecraft.util.registry.RegistryKey
+import net.minecraft.util.math.BlockPos
+
+class RtpCommand(val dispatcher: Dispatcher) {
+    private val rtpRange = -5000..5000
+
+    fun register() {
+        dispatcher.register(
+            CommandManager.literal("rtp")
+                .executes { rtpCommand(it) }
+        )
+        dispatcher.register(
```

Comment:
We don't do this - see @gdude2002's comment on my tpa PR

Reply:
I'm confused as to what not to do and can't seem to find his comment pertaining to registering commands, which is what I'm assuming this is about

Reply:
Oh, are you referring to him saying that each command should have its own class? /rtp and /wild are the same command, wild is just an alias for rtp, so I thought this was how to do that.

Reply:
It doesn't look like an alias - since it executes the same code. The way Minecraft does aliases is creating a `CommandNode` or whatever it's called (I don't remember Brigadier names) and using `.redirect` on the alias and passing in that node.

Reply:
Oh, well I didn't really know how to create a proper alias, but what I did is how the Fabric wiki showed how to do it. I can look into it and change it though

Reply:
```kotlin
fun register() {
        val node = registerMain()
        dispatcher.register(
            CommandManager.literal("wild")
                .redirect(node)
        )
    }

    fun registerMain(): LiteralCommandNode<ServerCommandSource> {
        return dispatcher.register(
            CommandManager.literal("rtp")
                .executes { rtpCommand(it) }
        )
    }
```
I did this based on what I could understand from the wiki. In game, it creates a /wild command, but when it is run, it gives the 'unknown command' error. Got any suggestions?